services/gcp/container/secrets.yml

```diff
@@ -28,17 +28,17 @@ privileges:
     risks:
       - discovery:data
       - discovery:infra
-      - escalation:lateral
-      - escalation:privilege
+      - exfiltration:crypto
+      - exfiltration:data
```

Comment:
Why is it also data exfiltration? Typical use would be to store "data" not in secrets manager.

Reply:
"Data exfiltration" here would cover all non-cryptographic uses of secrets manager. The "other sensitive data" referred to by Google in their product description. Honestly, I'm not sure exactly what that would be, and maybe it's not germane here 🤷